packages/koa-performance/README.md

```diff
@@ -0,0 +1,77 @@
+# `@shopify/koa-performance`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fkoa-performance.svg)](https://badge.fury.io/js/%40shopify%2Fkoa-performance.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/koa-performance.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/koa-performance.svg)
+
+A middleware which makes it easy to send metrics from your front end application and forward them to a stats-d server of your choice.
+
+Best used with [`@shopify/performance`](https://www.npmjs.com/package/@shopify/performance) and/or [`@shopify/react-performance`](https://www.npmjs.com/package/@shopify/react-performance).
+
+## Installation
+
+```bash
+$ yarn add @shopify/koa-performance
+```
+
+## Usage
+
+### Basic server setup
+
+First add the middleware to your Koa app. We recommend using `koa-mount` or `koa-router` to restrict it to a specific endpoint.
+
+```tsx
+// server.ts
+import Koa from 'koa';
+import mount from 'koa-mount';
+import {clientPerformanceMetrics} from '@shopify/koa-client-performance';
+
+const app = new Koa();
+
+app.use(mount('/client-metrics', clientPerformanceMetrics({
+  prefix: 'MyApp.',
+  // when development is true the middleware will skip sending metrics
+  development: process.env.NODE_ENV === 'development',
+  // the host of the statsd server you want to send metrics to
+  statsdHost: 'some-host',
+  // the port of the statsd server you want to send metrics to
+  statsdPort: 3000,
```

Comment:
```suggestion
  statsdPort: 8125,
```

Reply:
`react-server` default is 3000 plus 3000 is being use for the `app.listen`, lets avoid this